### PR TITLE
Add ha autoyast installation profile for sle15

### PR DESCRIPTION
**Pull request by chcao**
This pr is to add HA autoyast installation profile for SLES15SPx, for each arch, I add one profile.

- Related ticket: https://progress.opensuse.org/issues/116509; https://progress.opensuse.org/issues/116557
- Needles: n/a
- Verification run: 
autoyast installation and publish qcows cases:
15sp2: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP3&build=25.1&groupid=251
15sp3: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP3&build=25.1&groupid=251
15sp4: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=25.1&groupid=251
migration cases:
https://openqa.suse.de/tests/overview?distri=sle&version=15-SP5&build=25.1&groupid=251
(see the ha offline cases in this group, and all the ppc64le cases)

### CODE REVIEW

File: data/autoyast_sle15/create_hdd/create_hdd_ha_aarch64.xml.ep [hunk 1]
```diff
@@ -0,0 +1,326 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (lemon-suse) [inline data/autoyast_sle15/create_hdd/create_hdd_ha_aarch64.xml.ep:1]: This profile should support part of SLE15+ while not or SLE12+, so suggest to add 'sle15' at least in the names.

Comment (jknphy) [reply data/autoyast_sle15/create_hdd/create_hdd_ha_aarch64.xml.ep:1]: I think we made a mistake here @lemon-suse there was another folder with sle12, so we didn't need the prefix, like in this case :)

File: schedule/yast/autoyast/autoyast_ha.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+name: create_hdd_ha
+description: >
+  HA autoYaST installation and publish qcow for ha migration standalone
+  cases.
+vars:
+  AUTOYAST: autoyast_sle15/create_hdd/create_hdd_ha_%ARCH%.xml.ep
+  AUTOYAST_PREPARE_PROFILE: '1'
+  DESKTOP: textmode
+  SYSTEM_ROLE: HA_node
```
Comment (jknphy) [inline schedule/yast/autoyast/autoyast_ha.yaml:10]: probably we don't need this for auto-installation.

**Comment by jknphy** [comment]
I can see that there was used conditional scheduling, but I think for those modules to reboot the system is ok for now, we will tackle that in the future. Let's merge all and see result next week in dev group. Good job!

File: data/autoyast_sle15/create_hdd/create_hdd_ha_aarch64.xml.ep (hunk 1 above)
Comment (lemon-suse) [reply data/autoyast_sle15/create_hdd/create_hdd_ha_aarch64.xml.ep:1]: Yes, you are right, the folder is sle15. While in actual using for the profiles in the folder, we haven't distinguish much.(That is, even a case for SLE12+ we will use profile in the folder also.)

**Pull request merged**